phabricatoremails/service.py

```diff
@@ -107,5 +105,7 @@ def service(settings: Settings, stats: StatsClient):
     )
 
     render = Render(template_store)
-    pipeline = Pipeline(source, render, mail, logger, stats, settings.is_dev)
+    pipeline = Pipeline(
+        source, render, mail, logger, stats, settings.error_notify, settings.is_dev
```

Comment:
You're passing two parameters from `setting` here, perhaps passing `settings` and accessing what you need from it in `Pipeline` would be cleaner.